### ISSUE TITLE
Makefile not found and sidebar not activated

### ISSUE THREAD

**Issue by bht-patrick-schneider** [microsoft/vscode-makefile-tools#425]
Sorry for a potential dumb question. I read through the issues and tried a lot myself but I can't get the addon to work properly with my vscode.
I am on windows. My folder structure is as follows:
```
.vscode
app_example
   project_gcc
      Makefile
```

and I have the following settings in my settings.json in the .vscode folder

```
   "makefile.makefilePath": "${workspaceFolder}/app_example/project_gcc/Makefile",
   "makefile.loggingLevel": "Debug",
   "makefile.extensionLog": "${workspaceFolder}/.vscode/extension.log",
```

Whatever I do I get the error popup "The entry point Makefile was not found", when I manually start the configure task its the same.
The most weird behaviour is, that no logfile is created in the .vscode folder.
The sidebar items are not showing up either.
I tried reinstalling the addon but to no success.

**Comment by bobbrow** [comment]
I believe the problem is that we haven't finished and released the implementation for #343 yet.  The fix will be in 0.7.0, but for now, you can't use `${workspaceFolder}` in your makefile settings.

**Comment by bht-patrick-schneider** [comment]
That was exactly the problem. Thank you for pointing that out so quick @bobbrow.
